pandas/core/missing.py

```diff
@@ -404,7 +429,13 @@ def _from_derivatives(xi, yi, x, order=None, der=0, extrapolate=False):
     return m(x)
 
 
-def _akima_interpolate(xi, yi, x, der=0, axis=0):
+def _akima_interpolate(
+    xi: ArrayLike,
+    yi: ArrayLike,
+    x: Union[Scalar, ArrayLike],
+    der: Optional[int] = 0,
+    axis: Axis = 0,
```

Comment:
i think just int; `Axis` includes "index" and "columns" which dont get resolved here

Reply:
right, changed to int